src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -1237,7 +1237,7 @@ public static function closeOutputBuffers(int $targetLevel, bool $flush): void
      */
     protected function ensureIEOverSSLCompatibility(Request $request): void
     {
-        if (false !== stripos($this->headers->get('Content-Disposition'), 'attachment') && 1 == preg_match('/MSIE (.*?);/i', $request->server->get('HTTP_USER_AGENT'), $match) && true === $request->isSecure()) {
+        if (false !== stripos($this->headers->get('Content-Disposition') ?? '', 'attachment') && 1 == preg_match('/MSIE (.*?);/i', $request->server->get('HTTP_USER_AGENT') ?? '', $match) && true === $request->isSecure()) {
```

Comment:
Running this:
```
<?php
$haystack = "Hello World";
echo stripos ( $haystack, '');
?>
```
Echos zero.  Is this what you expect if the null coalescing operator executes and the empty string is used?  Test on https://extendsclass.com/php.html.  We are only looking for false, so I thought I fag the observed behaviour.

Reply:
`''` is the haystack, not the needle.

`?? ''` here provide the equivalent behavior than what you get with `null` in PHP 8.

Reply:
extendsclass.com don't have PHP 8.1. I don't get why to test ont it.